VideoWeb/VideoWeb/ClientApp/src/app/shared/select-media-devices/select-media-devices.component.scss

```
@@ -39,24 +43,30 @@
 }
 
 .wr-ml {
-  margin-left: 10px;
+  @media only screen and (min-width: $breakpoint-medium) {
+    margin-left: 10px;
+  }
+  @media only screen and (max-width: $breakpoint-medium) {
+    margin-top: -10px;
+  }
 }
 
 .wr-h {
   margin-left: 25px;
 }
 
-.camSwitchTitle {
-  margin-top: -25px;
-}
-
 // Rounded sliders
 .switch {
   position: relative;
   display: inline-block;
   width: 40px;
   height: 24px;
-  margin-bottom: 30px;
+  @media only screen and (max-width: $breakpoint-medium) {
+    margin-bottom: 0px;
+  }
+  @media only screen and (min-width: $breakpoint-medium) {
+    margin-bottom: 30px;
+  }
```

```suggestion
  margin-bottom: 30px;
  @media only screen and (max-width: $breakpoint-medium) {
    margin-bottom: 0px;
  }
```

One of these should be the default and we should only use either min or max for breakpoints. The example suggestion is desktop first but I want to see if anyone has any objections.